packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -82,6 +83,16 @@ export class AtomicQuickviewModal implements InitializableComponent {
 
   private interactiveResult?: InteractiveResult;
 
+  private get isMobile() {
```

Comment:
This is all available from the stencil store.

```typescript
this.bindings.store.isMobile()
```

Reply:
My problem with this is that it doesn't work for initializing a State variable since the store is not initialized yet.